### ISSUE TITLE
La cajita de ayuda de los envios no se puede cerrar

### ISSUE THREAD

**Issue by tvanessa** [omegaup/omegaup#2994]
Una alternativa es que solo aparezca una a la vez, otra alternativa es que solo aparezca el mensaje cuando el cursor este en la cajita de ayuda.
![Screen Shot 2019-10-27 at 1 20 46 PM](https://user-images.githubusercontent.com/13296248/67641126-f2a5c100-f8bc-11e9-9d28-329d9e19664b.png)

**Comment by pabloaguilar** [comment]
Puedes poner un poco mas de informacion, por favor?
Como version de browser, problema especifico y cosas asi.

Lo estoy intentando en los siguientes (donde tengo envios con diferentes tipos de evaluacion:
https://omegaup.com/arena/problem/Buscando-el-Ki#problems
https://omegaup.com/arena/problem/Multiplos-en-el-intervalo#problems
https://omegaup.com/arena/problem/Mario-rescata-a-Peach#problems
Y no lo puedor reproducir. Al darle click a otro boton de informacion la primer ventana desaparece y al darle click a cualquier otra parte la ventana tambien desaparece.

Esto con Chrome 77 en Windows.

**Comment by tvanessa** [comment]
Claro, esto ocurrió en los problemas del [curso OMI](https://omegaup.com/course/Curso-OMI), hice algunos envíos con errores en varios problemas y todos sobreponían la cajita de ayuda.

Estoy usando Firefox en Mac.

**Comment by pabloaguilar** [comment]
Gracias!
Lo intente con Firefox 68 y Firefox 70 en Windows y no lo puedo reproducir, al menos en los problemas que mencione arriba.
Voy a intentarlo en el curso de la OMI dado que puede que sea un problema exclusivo de schools.
Si no, supongo que es problema de FF en Mac

**Comment by pabloaguilar** [comment]
No lo puedo reproducir en el curso de la OMI tampoco.
Necesitamos alguien mas que tenga Mac que pueda reproducir esto.

**Comment by tvanessa** [comment]
Ya lo intente en Linux, y no pasa (Ni chrome ni firefox). 
Lo intente en la Mac de Hugo en Firefox y si se reproduce. En las mismas mac probe en Chrome y no se reproduce.
Voy a modificar los detalles del issue.
Gracias por las observaciones (:

**Comment by YannyTwoo** [comment]
Tried this on the recent Firefox, error does not show up.
Firefox 68 had a bug fixed which [didn't work](https://support.mozilla.org/en-US/questions/1267035) right so tooltips' pop-ups lingered on other tabs.
I couldn't recreate this on any other version from v101 to v110 so seems 100% a Firefox v68 issue. 
I think this issue can be closed.
